packages/core/echo/echo-schema/src/ref/ref.ts

```diff
@@ -291,7 +292,7 @@ export interface RefResolver {
   // TODO(dmaretskyi): Combine with `resolve`.
   resolveSchema(dxn: DXN): Promise<Schema.Schema.AnyNoContext | undefined>;
 }
-
+@trace.resource()
```

Comment:
no, this shouldn't be a resource: resource is to heavy for a ref. Ref is a lightweight API construct

you can put the span on database or hypergraph instead